app/src/pages/embedding/PointSelectionTable.tsx

```diff
@@ -186,6 +186,7 @@ export function PointSelectionTable({
     state: {
       sorting,
     },
+    columnResizeMode: "onChange",
```

Comment:
Is this change included here on purpose? Seems unrelated to the PR.

Reply:
you are right, I must have edited the wrong file